menas/src/main/scala/za/co/absa/enceladus/menas/repositories/RunMongoRepository.scala

```diff
@@ -53,6 +55,51 @@ class RunMongoRepository @Autowired()(mongoDb: MongoDatabase)
     excludeId()
   ))
 
+  private def getTodaysFilter() = {
+    val date = ZonedDateTime.now().format(DateTimeFormatter.ofPattern("dd-MM-yyyy"))
+    regex("startDateTime", s"^$date")
+  }
+
+  private def getTodayRunsCount(filters: List[Bson]): Future[Int] = {
+    val pipeline = Seq(
+      filter(and((getTodaysFilter :: filters) :_*)),
+      Aggregates.count("count"))
+    collection.aggregate[BsonDocument](pipeline).headOption().map({
+        case Some(doc) => doc.getInt32("count").getValue
+        case None => 0
+    })
+  }
+
+  def getTodaysRuns(): Future[Int] = {
+    getTodayRunsCount(List())
+  }
+
+  def getTodaysSuccessfulRuns(): Future[Int] = {
+    getTodayRunsCount(List(Filters.eq("runStatus.status", "allSucceeded")))
+  }
+
+  def getTodaysFailedRuns(): Future[Int] = {
+    getTodayRunsCount(List(Filters.eq("runStatus.status", "failed")))
+  }
+
+  def getTodaysStdSuccessRuns(): Future[Int] = {
+    getTodayRunsCount(List(Filters.eq("runStatus.status", "stageSucceeded")))
+  }
+
+  def getTodaysRunningRuns(): Future[Int] = {
+    getTodayRunsCount(List(Filters.eq("runStatus.status", "running")))
+  }
+
+  def getTodaysSuccessWithErrors(): Future[Int] = {
+    getTodayRunsCount(List(
+        Filters.eq("runStatus.status", "allSucceeded"),
+        or(
+        and(Filters.exists("controlMeasure.metadata.additionalInfo.std_errors_count"),
+            Filters.notEqual("controlMeasure.metadata.additionalInfo.std_errors_count", "0")),
+        and(Filters.exists("controlMeasure.metadata.additionalInfo.std_errors_count"),
```

Comment:
```suggestion
        and(Filters.exists("controlMeasure.metadata.additionalInfo.conform_errors_count"),
```

Reply:
this will then require a change of the names of the columns in graphs.

Reply:
> this will then require a change of the names of the columns in graphs.

Why?

Reply:
We are speaking about standardization there. Aren't we? I don't have it built currently. 
Should we speak about conformance?